client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/db/mongo/MongoDistanceWithMetrics.java

```diff
@@ -0,0 +1,40 @@
+package org.evomaster.client.java.controller.internal.db.mongo;
```

Comment:
files should be "moved" and not "deleted/added", as in this latter case we would lose git history, and makes harder to check if introduced any error by mistake